CHANGELOG.md

```diff
@@ -35,6 +35,7 @@ We use *breaking :warning:* to mark changes that are not backward compatible (re
 - [#7853](https://github.com/thanos-io/thanos/pull/7853) UI: Add support for selecting graph time range with mouse drag.
 - [#7855](https://github.com/thanos-io/thanos/pull/7855) Compcat/Query: Add support for comma separated replica labels.
 - [#7654](https://github.com/thanos-io/thanos/pull/7654) *: Add '--grpc-server-tls-min-version' flag to allow user to specify TLS version, otherwise default to TLS 1.3
+- [#7366](https://github.com/thanos-io/thanos/pull/7366) Replicate: Add flag to mark blocks for deletion after replication
```

Comment:
Can you please mention the flag name in the changelog so that users don't have to go and look up which flag does this

Reply:
done